src/jdk.jdwp.agent/share/native/libjdwp/util.c

```diff
@@ -34,7 +40,10 @@
 #include "inStream.h"
 #include "invoker.h"
 #include "signature.h"
-
```

Comment:
Please leave this line from upstream alone.

Reply:
Restore the empty line.